src/auth/interceptors/createJwtTokenProviderInterceptor.js

```diff
@@ -0,0 +1,41 @@
+import getJwtToken from '../getJwtToken';
+
+const createJwtTokenProviderInterceptor = (options) => {
```

Comment:
Renamed to createJwtTokenProviderInterceptor from jwtTokenProviderInterceptor to better indicate what is happening here.